tool/update_bindings.dart

```diff
@@ -0,0 +1,125 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:args/args.dart';
+import 'package:io/ansi.dart' as ansi;
+import 'package:io/io.dart';
+import 'package:path/path.dart' as p;
+
+void main(List<String> arguments) async {
+  final ArgResults argResult;
+
+  try {
+    argResult = _parser.parse(arguments);
+  } on FormatException catch (e) {
+    print('''
+${ansi.lightRed.wrap(e.message)}
+
+Usage:
+${_parser.usage}''');
+    exitCode = ExitCode.usage.code;
+    return;
+  }
+
+  assert(p.fromUri(Platform.script).endsWith(_thisScript));
+
+  // npm install
```

Comment:
Could you make the comments in this file complete sentences that start with capital letters and end with periods?

i.e. `Install npm.

Reply:
sure?